src/lib/Counter.svelte

```diff
@@ -2,17 +2,21 @@
 	import { getStore } from '$lib/utils/hmr-stores';
 
 	export let id: string;
+	export let agent: string;
 
 	const count = getStore(id, 0);
 
 	const handleClick = () => {
 		$count += 1;
-		window.api.send('to-main', $count);
 	};
+
+	if (window.electron) {
+		count.subscribe((c) => window.electron.send('to-main', c));
+	}
```

Comment:
This should fix the unsub issue

```html
$: if (window.electron) {
    window.electron.send('to-main', $count)
}
```

Reply:
@ghostdevv 

Sorry for the slow reply, have been busy. Happy to implement your change, but there is a slight difference between subscribing with `$: ` and `.subscribe()`.

https://svelte.dev/repl/d595a50133a64f52936a66b9b645a685?version=3.47.0

I can either implement the `$: ` as you suggested, which is definitely cleaner, or I can keep using the more accurate `.subscribe()`, and fix the unsub issue by importing `onDestroy`? What would you prefer?

Reply:
From looking at the compiled output the reason is that since the two statements in doubleUpdate are directly next to each other svelte will not have processed the dom updates by the time the second statement is ran so it will only know about the final change

I don't see this being an issue myself but if you have a case where it is then I would be interested to see it

Reply:
Yeah, you're right - I've amended my PR implementing your initial suggestion :slightly_smiling_face: